storage/io/src/test/java/org/openscience/cdk/io/MDLV3000ReaderTest.java

```diff
@@ -117,4 +125,13 @@ public void testPseudoAtomLabels() throws Exception {
         }
     }
 
+    @Test public void positionalVariation() throws Exception {
+        MDLV3000Reader reader = new MDLV3000Reader(getClass().getResourceAsStream("multicenterBond.mol"));
+        IAtomContainer container = reader.read(new org.openscience.cdk.AtomContainer(0,0,0,0));
```

Comment:
No I should have closed it there. The main problem is you can only have a certain number of file pointers open on the OS.